frontend/src/components/FooTouer/FooTouer.tsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import './FooTouer.css';
+import MyButton from "../Button/Button";
+import ButtonComponent from "../Button/Button";
+
 
 function FooTouer() {
-  return (
-    <div className="FooTouer">
-      HelloWorld
-    </div>
-  );
+    return (
+        <>
+            <div className="FooTouer">
+                HelloWorld
+            </div>
+            {/*<Button name={"Hello button"} onClick={()=>alert(`Hi, I'm clicked :)))`)}/>*/}
+            {ButtonComponent({name: "Hello button", onClick: (() => alert(`Hi, I'm clicked!`)),})}
```

Comment:
tak się robi :-) ?
Bo na tutorialach wgl nie widziałem takich zapisów, stąd pytanie.
Było coś w deseń 
`<MyButton onClick={props.onClick}>{props.name}</MyButton>` ale już w klasie rodzica, a tutaj jest wywołanie funkcji

Reply:
O ludzie, ciągle się czegoś od Was uczę :D 
A bardziej challenguje swoją wiedzę. Nie spodziewałem się, że to w ogóle zadziała. 
W życiu bym na to nie wpadł, żeby tak umieszczać komponent :) 

Wystarczy zostawic tylko jak wyzej. 

Jakies dodatkowe zrodla:

https://kentcdodds.com/blog/dont-call-a-react-function-component
https://medium.com/swlh/never-call-a-react-function-component-6815a92b7856
https://dev.to/igor_bykov/react-calling-functional-components-as-functions-1d3l